src/set_theory/cardinal.lean

```diff
@@ -645,6 +645,76 @@ by rw [← succ_zero, succ_le]
 theorem one_le_iff_ne_zero {c : cardinal} : 1 ≤ c ↔ c ≠ 0 :=
 by rw [one_le_iff_pos, pos_iff_ne_zero]
 
+section bit
+-- This section proves the "easy" inequalities for `bit0` and `bit1`,
+-- enabling `simp` to solve `0 < n`, `0 ≤ n`, `1 < n` and `1 ≤ n` for numerals `n`.
+
+-- See `one_lt_two` for a general tactic solution for proving inequalities between numerals
+-- using the embedding of `ℕ`.
+
+-- To enable `simp` to solve all inequalities between numerals we'd need other results,
+-- e.g. beginning with `bit0 a ≤ bit0 b → a ≤ b`.
```

Comment:
This looks like it could be useful text to have in the mathlib docs.
```suggestion
/-! 
This section proves the "easy" inequalities for `bit0` and `bit1`,
enabling `simp` to solve `0 < n`, `0 ≤ n`, `1 < n` and `1 ≤ n` for numerals `n`.

See `one_lt_two` for a general tactic solution for proving inequalities between numerals
using the embedding of `ℕ`.

To enable `simp` to solve all inequalities between numerals we'd need other results,
e.g. beginning with `bit0 a ≤ bit0 b → a ≤ b`.
-/
```